demo/text_matching/run_predict.sh

```diff
@@ -0,0 +1,15 @@
+export FLAGS_eager_delete_tensor_gb=0.0
```

Comment:
Please remove all of those FLAGS_eager_delete_tensor_gb of the demo code.
Since in PaddlePaddle 1.8, this is a default setting.
Simplify scripts.